metricbeat/mb/testing/data_generator.go

```diff
@@ -75,13 +75,13 @@ func WriteEventsCond(f mb.EventsFetcher, t testing.TB, cond func(e common.MapStr
 	}
 
 	fullEvent := CreateFullEvent(f, *event)
-	WriteEventToDataJSON(t, fullEvent)
+	WriteEventToDataJSON(t, fullEvent, "")
 	return nil
 }
 
 // WriteEventsReporterV2 fetches events and writes the first event to a ./_meta/data.json
 // file.
-func WriteEventsReporterV2(f mb.ReportingMetricSetV2, t testing.TB) error {
+func WriteEventsReporterV2(f mb.ReportingMetricSetV2, t testing.TB, path string) error {
```

Comment:
I wonder if the path is going to be most of the times based on the metricset name, because then maybe we could guess it from `f.Name()` or similar. This would require less knowlegde when writting tests. WDYT?

Reply:
It is for all tests which are written on the module level. Not sure yet where we end up here TBH. Ok to simplify this later?